R/binary_segmentation.R

```diff
@@ -218,7 +220,7 @@ FindBestSplit <- function(x, start, end, delta, n_obs, control, SegmentLossFUN,
     min(obs_count - 1, obs_count - min_seg_length + 1), 1
   )
 
-  segment_loss <- SegmentLossFUN(x, start, end)
+  segment_loss <- SegmentLossFUN(x) #, start, end)
```

Comment:
This will break my code for the change in mean stuff. We need to solve this globally like you suggest in issue #34